qtop.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
 ################################################
-#              qtop v.0.8.6                    #
+#              qtop v.0.8.7                    #
```

Comment:
wow, one of the nicest PRs ever.
to be merged. remember the need for a refreshed changelog!